networkx/algorithms/min_edge_cover.py

```diff
@@ -0,0 +1,74 @@
+"""
+An edge cover of a graph is a set of edges such that every vertex of
+the graph is incident to at least one edge of the set.
+A minimum edge cover is an edge covering of smallest possible size.
+
+https://en.wikipedia.org/wiki/Edge_cover
+"""
+#    Copyright (C) 2015 by
+#    Nishant Nikhil <nishantiam@gmail.com>
+#    All rights reserved.
+#    BSD license.
+
+import networkx as nx
+from networkx.algorithms import bipartite
+
+
+def min_edge_cover(G):
+    """
+    A smallest edge cover can be found in polynomial time by finding
+    a maximum matching and extending it greedily so that all vertices
+    are covered.
+
+    Parameters
+    ----------
+    G : NetworkX graph
+        Undirected graph
+        Unweighted graph
+
+    Returns
+    -------
+    min_cover : dictionary
+        The matching is returned as a dictionary, min_weighted_edge_cover,
```

Comment:
Code blocks must be styled as such, using double backticks:

    ``min_weighted_edge_cover``